build/Makefile

```diff
@@ -421,8 +426,10 @@ endif
 
 build-controller-image-amd64: $(ensure-build-image) build-controller-binary build-licenses build-required-src-dist
 	docker build $(agones_path)/cmd/controller/ --tag=$(controller_amd64_tag) $(DOCKER_BUILD_ARGS)
+# creating docker builder and then using that builder to build controller image in buildx command
 build-controller-image-arm64: $(ensure-build-image) build-controller-binary
-	docker buildx build --platform linux/arm64 --build-arg ARCH=arm64 $(agones_path)/cmd/controller/ --tag=$(controller_arm64_tag) $(DOCKER_BUILD_ARGS)
+	$(MAKE) create-arm64-builder 
```

Comment:
Should this be:

`build-controller-image-arm64: $(ensure-build-image) build-controller-binary create-arm64-builder`

Rather than calling `create-arm64-builder` within the `build-controller-image-arm64` target?

(And same for other targets as well)

Reply:
Thanks for the suggestion, Yes it works 😃.